library-valcon-internship/src/components/Header/Header.css

```diff
@@ -0,0 +1,90 @@
+.Header {
```

Comment:
Maybe it would be good idea to have all class names written  with first letter lowercase?